packages/flutter_tools/lib/src/asset.dart

```diff
@@ -50,6 +50,8 @@ class AssetBundle {
     }
   }
 
+  bool wasBuildOnce() => _lastBuildTimestamp != null;
```

Comment:
nit: "wasBuiltOnce" is more gramatically correct, I think?

Reply:
Fixed.